client/src/assets/custom.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module "*.jpg";
```

Comment:
I am not sure, but why do we need this file? Webpack configs which created with create-react-app perfectly fine may recognize those image types

Reply:
he don't see my image